backend/prompt_studio/prompt_studio_registry/prompt_studio_registry_helper.py

```diff
@@ -41,7 +42,40 @@ def frame_spec(tool: CustomTool) -> Spec:
         Returns:
             dict: spec dict
         """
-        spec = Spec(title=str(tool.tool_id), description=tool.description)
+        properties = {
+            "challenge_llm": {
+                "type": "string",
+                "title": "Challenge LLM",
+                "adapterType": "LLM",
+                "description": "LLM to use for challenge",
+                "adapterIdKey": "challenge_llm_adapter_id",
+            },
+            "enable_challenge": {
+                "type": "boolean",
+                "title": "Challenge",
```

Comment:
I will change it to `Enable challenge` to match the name in Challenge manager. I thought enable makes it redundant because we are having a checkbox that enables it. We can properly fix it along with the references separately,